radio/src/targets/simu/opentxsimulator.cpp

```diff
@@ -236,16 +239,23 @@ void OpenTxSimulator::setTrainerInput(unsigned int inputNumber, int16_t value)
 
 void OpenTxSimulator::setInputValue(int type, uint8_t index, int16_t value)
 {
-  //qDebug() << type << index << value << this->thread();
+  qDebug() << type << index << value << this->thread();
```

Comment:
I think this should be disabled again

Reply:
Agree but still working on trims and there will be more like this until resolved. Changed PR to Merge state so all libsims compile